app/services/bulk_api_import/fhir_importable.rb

```diff
@@ -0,0 +1,24 @@
+module BulkApiImport::FhirImportable
+  def import_user
+    ImportUser.find_or_create
+  end
+
+  def import
+    raise NotImplementedError
+  end
```

Comment:
leaving this here to signal that any class that includes this mixin is expected to have an `import` method.